salt/utils/http.py

```diff
@@ -70,12 +70,12 @@
 # pylint: enable=import-error,no-name-in-module
 
 # Don't need a try/except block, since Salt depends on tornado
-import tornado.httputil
-import tornado.simple_httpclient
-from tornado.httpclient import HTTPClient
+import salt.ext.tornado.httputil
+import salt.ext.tornado.simple_httpclient
+from salt.ext.tornado.httpclient import HTTPClient
```

Comment:
Should `import salt.ext.tornado.httpclient` and `salt.ext.tornado.httpclient.HTTPClient` be used instead?

Reply:
I'm not making any code cleanup in this PR to keep down on the amount of change. This should get fixed at some point in the future though.